forge-gui/res/cardsfolder/upcoming/hurkyl_master_wizard.txt

```diff
@@ -0,0 +1,24 @@
+Name:Hurkyl, Master Wizard
+ManaCost:1 U U
+Types:Legendary Creature - Human Wizard Advisor
+PT:2/4
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | SVarCompare$ GE1 | Execute$ TrigDig | TriggerDescription$ At the beginning of your end step, if you've cast a noncreature spell this turn, reveal the top five cards of your library. For each card type among noncreature spells you've cast this turn, you may put a card of that type from among the revealed cards into your hand. Put the rest on the bottom of your library in a random order.
+SVar:TrigDig:DB$ Dig | DigNum$ 5 | Reveal$ True | NoMove$ True | RememberRevealed$ True | SubAbility$ DBChooseCardOne
+SVar:DBChooseCardOne:DB$ ChooseCard | Defined$ You | Choices$  Instant.YouCtrl+IsRemembered  | ImprintChosen$ True | ConditionCheckSVar$ CheckInstant | ConditionSVarCompare$ EQ1 | ChoiceTitle$ You may choose an instant card if you cast an instant spell this turn | ChoiceZone$ Library | AILogic$ BestCard | SubAbility$ DBChooseCardTwo 
```

Comment:
the choices should be optional?

Reply:
I think with Choosecard the default is optional example 
![image](https://user-images.githubusercontent.com/67333662/198841270-751a835b-bf2c-4cc2-b26b-90b4d585f992.png)